src/hdf5/atms_netcdf_hdf5_2ioda.py

```diff
@@ -72,51 +86,37 @@ def main(args):
             concat_obs_dict(obs_data, file_obs_data)
         else:
             obs_data = file_obs_data
+        if WMO_sat_ID != GlobalAttrs['platform']:
+            print(' IODA and subsequent UFO expect files to be by satellite and sensor ')
```

Comment:
Is this an error condition that warrants stopping execution? If so it should raise an Exception or exit and it would be helpful to indicate in the message that this is an error. If not, the message should indicate that this is a warning. Thanks!

Reply:
Steve added  `ERROR` fixed the check as it would have never hit

and added an exit